java/src/jmri/jmrit/display/layoutEditor/MultiSensorIconFrame.java

```diff
@@ -164,11 +176,11 @@ public String toString() {
         Entry(JPanel self, JmriJFrame frame, String name) {
             this.self = self;
             this.setLayout(new FlowLayout());
-            this.add(new JLabel("Sensor:"));
+            this.add(new JLabel(Bundle.getMessage("BeanNameSensor") + ":"));
```

Comment:
Method 1 is the recommended practice in most situations--this is all about having flexibility of context. It's entirely possible that in some language, text for "Thing" when used in a widget label is not the same as the text for Thing in another context. Even you do method 2, you will find some translations will not work without reverting to method 1.